modules/ROOT/pages/emergency-shell.adoc

```diff
@@ -1,14 +1,52 @@
-= Access the emergency shell
+= Emergency console access
 
+Sometimes you may want to access the node console to perform troubleshooting steps or emergency maintenance.
+For instance, you may want to access the emergency shell on the console, in order to debug first boot provisioning issues.
```

Comment:
I feel like we should mention somewhere around here the semantics that the last console arg wins which is what motivates wanting to modify the kargs when trying to access the emergency console.

Reply:
Ack. I double-checked how this works with https://www.kernel.org/doc/html/v4.14/admin-guide/serial-console.html, then added a second commit with some more details.